public/js/controllers/index.js

```diff
@@ -166,10 +178,23 @@ angular.module('mean.system').controller('IndexController', [
       messageArray.push(message);
       $scope.notifications = messageArray;
       $scope.messageLength = messageArray.length;
+      console.log('@@@@@@@@@@@',messageArray);
```

Comment:
Here too. Please remove console statement.

Reply:
I have done that